Impossible to run on OSX (shader)

Latest version on git:

./bin/Ikemen_GO_MacOS                                                                                                                                   ✔  8461  10:32:52
2022/11/22 10:32:54 Using OpenGL 2.1 ATI-4.8.101 (AMD Radeon Pro Vega 56 OpenGL Engine)
panic: Shader compile error: ERROR: 0:1: '' :  version '400' is not supported
ERROR: 0:2: 'precision' : syntax error: syntax error

goroutine 1 [running, locked to thread]:
main.compileShader(0x20d500?, {0x1004bf355, 0xe4})
	github.com/ikemen-engine/Ikemen-GO/src/render_gl.go:86 +0x14d
main.newShaderProgram({0x1004bf355?, 0xc000000001?}, {0x1004bfa80, 0x54e}, {0xc0000061a0?, 0x100006dff?})
	github.com/ikemen-engine/Ikemen-GO/src/render_gl.go:51 +0x3d
main.(*Renderer).Init(0x10077f3c0)
	github.com/ikemen-engine/Ikemen-GO/src/render_gl.go:201 +0x310
main.(*System).init(0x100771000, 0x18?, 0x0?)
	github.com/ikemen-engine/Ikemen-GO/src/system.go:406 +0x285
main.main()
	github.com/ikemen-engine/Ikemen-GO/src/main.go:83 +0x19f

Seems related to last changement on the shader by @samhocevar  ?

Thx

--- a/src/render_gl.go
+++ b/src/render_gl.go
@@ -7,6 +7,7 @@ import (
 	"encoding/binary"
 	"fmt"
 	"runtime"
+	"strings"
 
 	gl "github.com/fyne-io/gl-js"
 	"golang.org/x/mobile/exp/f32"
@@ -77,6 +78,10 @@ func (s *ShaderProgram) RegisterTextures(names ...string) {
 
 func compileShader(shaderType gl.Enum, src string) (shader gl.Shader) {
 	shader = gl.CreateShader(shaderType)
+	// Might be necessary for the WebGL build
+	if strings.Contains(gl.GetString(gl.VERSION), "ES") {
+		src = "#version 100\nprecision highp float;\n"
+	}
 	gl.ShaderSource(shader, src)
 	gl.CompileShader(shader)
 	ok := gl.GetShaderi(shader, gl.COMPILE_STATUS)

--- a/src/shaders/ident.frag.glsl
+++ b/src/shaders/ident.frag.glsl
@@ -1,10 +1,7 @@
-#version 400
-precision highp float;
-
 uniform sampler2D Texture;
 
-in vec2 texcoord;
+varying vec2 texcoord;
 
 void main(void) {
-	gl_FragColor = texture(Texture, texcoord);
+	gl_FragColor = texture2D(Texture, texcoord);
 }

--- a/src/shaders/ident.vert.glsl
+++ b/src/shaders/ident.vert.glsl
@@ -1,10 +1,8 @@
-#version 400
-precision highp float;
+attribute vec2 VertCoord;
 
 uniform vec2 TextureSize;
 
-in vec2 VertCoord;
-out vec2 texcoord;
+varying vec2 texcoord;
 
 void main()
 {

--- a/src/shaders/sprite.frag.glsl
+++ b/src/shaders/sprite.frag.glsl
@@ -1,6 +1,3 @@
-#version 400
-precision highp float;
-
 uniform sampler2D tex;
 uniform sampler2D pal;
 
@@ -11,7 +8,7 @@ uniform float alpha, gray;
 uniform int mask;
 uniform bool isFlat, isRgba, isTrapez, neg;
 
-in vec2 texcoord;
+varying vec2 texcoord;
 
 void main(void) {
 	if (isFlat) {
@@ -26,7 +23,7 @@ void main(void) {
 			uv.x = (gl_FragCoord.x - bounds[0]) / (bounds[1] - bounds[0]);
 		}
 
-		vec4 c = texture(tex, uv);
+		vec4 c = texture2D(tex, uv);
 		vec3 neg_base = vec3(1.0);
 		vec3 final_add = add;
 		vec4 final_mul = vec4(mult, alpha);
@@ -40,7 +37,7 @@ void main(void) {
 			if (int(255.25*c.r) == mask) {
 				final_mul = vec4(0.0);
 			} else {
-				c = texture(pal, vec2(c.r*0.9966, 0.5));
+				c = texture2D(pal, vec2(c.r*0.9966, 0.5));
 			}
 		}
 

--- a/src/shaders/sprite.vert.glsl
+++ b/src/shaders/sprite.vert.glsl
@@ -1,11 +1,8 @@
-#version 400
-precision highp float;
-
 uniform mat4 modelview, projection;
 
-in vec2 position;
-in vec2 uv;
-out vec2 texcoord;
+attribute vec2 position;
+attribute vec2 uv;
+varying vec2 texcoord;
 
 void main(void) {
 	texcoord = uv;

